make/common/JdkNativeCompilation.gmk

```diff
@@ -303,6 +303,12 @@ define SetupJdkNativeCompilationBody
     $1_RC_FTYPE := 0x2L
   endif
 
+  ifneq ($$(MODULE), )
+    ifneq ($$($1_EXCLUDE_FROM_STATIC_LIBS), true)
+      $$(MODULE)_JDK_LIBS += $$($1_NAME)
+    endif
+  endif
```

Comment:
Same, here as in ModuleWrapper.gmk, I think we need a comment explaining how this is consumed.

Reply:
Fixed.